data/json/items/appliances.json

```diff
@@ -320,5 +320,43 @@
     "looks_like": "battery_charger",
     "flags": [ "NO_SALVAGE" ],
     "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "50 L", "max_contains_weight": "100 kg" } ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "compact_ASRG_containment",
+    "name": { "str_sp": "Shielded Compact Advanced Sterling Radioisotope Generator" },
+    "looks_like": "t_plut_generator",
+    "description": "A really heavy brick of metal housing an ASRG, formerly used by NASA and repurposed by the military.  Will provide about 130w, basically forever as far as you're concerned.  Should be safe due to the large amount of shielding.  Although these were designed for spacecraft, the sheer amount of armor plating and lead means it is much more massive, so you probably shouldn't try to power a vehicle with it.",
+    "weight": "400 kg",
+    "to_hit": -10,
+    "material": [ "lead", "steel" ],
+    "volume": "300 L",
+    "longest_side": "100 cm",
+    "bashing": 5,
+    "//": "I don't know how much these were worth pre-Cataclysm but the pu-238 alone is worth about 5-6 million and hundreds of millions were spent on research.  After the apocalypse these should basically be valuable beyond compare though.",
+    "price": 1000000000,
+    "price_postapoc": 2500000,
+    "symbol": "O",
+    "color": "green_white",
+    "flags": [ "LEAK_DAM" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "compact_ASRG",
+    "name": { "str_sp": "Unshielded Compact Advanced Sterling Radioisotope Generator" },
+    "looks_like": "t_plut_generator",
+    "description": "A moderately heavy brick of metal housing an ASRG.  Most of the lead and steel plating has been salvaged off so it's a lot less bulky and heavy.  However it's a lot less sturdy, so the threat of a radioactive breach is much more real.",
+    "weight": "60 kg",
+    "to_hit": -10,
+    "material": [ "lead", "steel" ],
+    "volume": "75 L",
```

Comment:
The volume could well be even larger actually; even the listed prototype dimensions give you 136.3 L, assuming it's a rectangular cuboid.